src/meta/MetaServiceUtils.cpp

```diff
@@ -318,11 +376,21 @@ HostAddr MetaServiceUtils::parseLeaderKeyV1(folly::StringPiece key) {
 }
 
 HostAddr MetaServiceUtils::parseLeaderKeyV2(folly::StringPiece key) {
+    LOG(FATAL) << "deprecated function";
```

Comment:
There is a problem here,
 if upgrade. Is the previous meta leader data invalid?
How to delete?